stackslib/src/chainstate/stacks/db/blocks.rs

```diff
@@ -560,6 +560,28 @@ impl StacksChainState {
         Ok(inst)
     }
 
+    pub fn consensus_load_with_epoch<T: DeserializeWithEpoch>(
```

Comment:
Stacks blocks are only stored to disk up through epoch 2.5, so this is not needed